install_sct

```diff
@@ -113,13 +115,14 @@ function finish() {
   cd "$SCT_SOURCE"
   if [[ "$value" -eq 0 ]]; then
     print info "Installation finished successfully!"
+    rm -r install_sct_log.txt  # if there's no errors, there's no need to keep this log around
   elif [[ "$value" -eq 99 ]]; then
     # Showing usage with -h
     echo ""
   else
     print error "Installation failed!\n
-Please copy the output of this Terminal (starting with the command install_sct) and upload it as a .txt attachment in a new topic on SCT's forum:\n
---> http://forum.spinalcordmri.org/c/sct"
+Please find the file $(pwd)/install_sct_log.txt and \n
+attach it to a new issue in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/new so we can help you."
```

Comment:
This bumps into the question of "What do we want to use issues for, and what do we want to use the forum for?"

I think currently, it's something like:

* GitHub: All dev activity.
* Forum: All user activity.

But the proposed is:

* GitHub: Installation issues, dev activity
* Forum: Usage issues/questions

I _think_ I like the proposed split? It basically means that GitHub becomes "the place where broken things get reported and fixed", and removes some clutter from the forum, letting it stay focused on: "OK, I have a working installation. Now, how do I use SCT effectively?"

Reply:
hum... on the other hand, we are dealing with users who can easily get overwhelmed with +1 platform to deal with.

Reply:
I figured this line would be controversial :P. No worries, we can back it out.

Before we do though, can we take a survey? Of people who've had installation problems in the last year or so, people whose first post(s) were about it:

* https://forum.spinalcordmri.org/u/aran/activity
* https://forum.spinalcordmri.org/u/davidcadotte/activity
* https://forum.spinalcordmri.org/u/annekbaker/activity
* https://forum.spinalcordmri.org/u/abdullah_althubaiti/activity
* https://forum.spinalcordmri.org/u/alexander_zhang/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/b_zeinali/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/salmanshaz/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/hussnain/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/vb922/activity (installation are their only posts)
* https://forum.spinalcordmri.org/u/mahdi/activity
* https://forum.spinalcordmri.org/u/deng/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/jetpack/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/willk/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/pelegrini/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/gmotzespina/activity
* https://forum.spinalcordmri.org/u/gali/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/sahar/activity
* https://forum.spinalcordmri.org/u/idris/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/mvaro2002/activity
* https://forum.spinalcordmri.org/u/felixi/activity
* https://forum.spinalcordmri.org/u/jtalbott/activity
* https://forum.spinalcordmri.org/u/amijas/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/deshengkong/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/spine/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/brma/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/marco/activity
* https://forum.spinalcordmri.org/u/georgebouz/activity (installation is their only post)
* https://forum.spinalcordmri.org/u/miki/activity (installation is their only post; but deviating from the normal bug report, [this](https://forum.spinalcordmri.org/t/install-previous-versions/240) is actually #3583, but I'd still want that in the bug tracker)
* https://forum.spinalcordmri.org/u/petermultivac/activity


Of them, people whose weren't:

* https://forum.spinalcordmri.org/u/rosella_tro/activity
* https://forum.spinalcordmri.org/u/zmzforever/activity
* https://forum.spinalcordmri.org/u/kim/activity (however, their *second* post was, and their first was about a different code issue)
* https://forum.spinalcordmri.org/u/chang/activity
* https://forum.spinalcordmri.org/u/hanna/activity
* https://forum.spinalcordmri.org/u/weizx/activity
* https://forum.spinalcordmri.org/u/daniel_papp/activity
* https://forum.spinalcordmri.org/u/ihattan/activity
* https://forum.spinalcordmri.org/u/merve/activity

I think this shows that people are generally _creating_ accounts to report bugs. Even amongst the people who weren't, a couple only logged back in after a couple of years because they hit a problem while upgrading SCT.

A couple of the issues were user oversights like forgetting `./` in `./install_sct`, but most of them were legitimate issues. Even the user oversights are bug-ish since they're opportunities to clarify our documentation. So, I hate to be "that person" because I'm not a huge fan of Github in principle, but my thinking was, if they need to create an account anyway, it might as well be here with the rest of the bugs.

Reply:
A different approach entirely: we could (with consent) automatically upload the failed installation logs somewhere, and have a bot post them to Github for us with a `triage` tag.

But then I guess we leave the user without any recourse to tech support :thinking:

Reply:
i do see the value of removing the 'noise' in the forum. On the other hand, I'm afraid to see the other effect: topics that should be in the forum (ie: useful for the community) be put on GH. 

having the clear distinction: GH is for the devs | Forum is for users, makes it less confusing

Reply:
FWIW, I think it's probably safe enough to stick with what we're doing anyway, rather than rock the boat (especially for a PR that's mainly about generating a log file.)

So, I'm :+1: on not changing anything here.